samples/net/full_mqtt_tls/src/dhcp.c

```diff
@@ -0,0 +1,87 @@
+/* DHCPv4 client startup. */
```

Comment:
Is this file used? I thought dhcp was being handled by the net app layer.

Reply:
no, it still has to be kicked off here.  from what i can tell at least.